bluepymm/select_combos/megate_output.py

```diff
@@ -99,7 +98,11 @@ def save_megate_results(extneurondb, output_dir,
                 #        extneurondb[null_rows]))
                 print("WARNING ! There are rows with None for "
                       "holding current: %s" % str(extneurondb[null_rows]))
+                extneurondb = extneurondb[
```

Comment:
Is this not going to mess up the reports? I assume these deleted rows are not removed from the reports? Or is the report generated afterwards?

Reply:
the report is made before I think: https://github.com/BlueBrain/BluePyMM/blob/master/bluepymm/select_combos/main.py

Reply:
Ok, so are we sure that this is tested before the reports are made? Because otherwise, if we remove the rows automatically, we need to make sure these combos don't appear in the report as 'succeeded'